Python/quickstarts/callable-functions/functions/main.py

```diff
@@ -15,9 +15,14 @@
 import re
 from typing import Any
 
-from firebase_admin import db, initialize_app
+# [START v2imports]
+# Dependencies for callable functions
 from firebase_functions import https_fn, options
 
+# Dependencies for messageFunctionTrigger
```

Comment:
How about "Dependencies for writing to RTDB"?

(There's no messageFunctionTrigger)

Reply:
Got it!  I think I stole that from a tag name :)